doc/starting-guide.md

```diff
@@ -4,21 +4,46 @@ This document provides information on how to start developing Status App.
 
 # Getting Started
 
-To start developing start a shell for the platform you are interested in.
+To start developing clone the status-mobile repo in the directory of your choice. 
 ```
-make shell TARGET=android
+git clone https://github.com/status-im/status-mobile.git
 ```
-This step will take a while the first time as it will download all dependencies.
 
-# Development
+Then open a terminal and cd into this directory. 
 
-There are three steps necessary to start development, in this case for Android:
+```
+cd status-mobile
+```
+
+Then build the clojure terminal 
+
+```
+make run-clojure
+```
 
-1. `make run-clojure` - Compiles Clojure into JavaScript, watches for changes on cljs files, and hot-reloads code in the app.
-2. `make run-android` or `make run-ios` - Builds the Android/iOS app, starts it on the device and starts metro bundler.
+note: ⚠️ This might take a while if this is your first time. 
 
-The first two will continue watching for changes and keep re-building the app. They need to be ready first.
-The last one will exit once the app is up and ready.
+This command installs `nix` and pulls in all the dependencies.
+
+Do answer with "Y" to all the prompts and press "Enter" when `nix` setup asks you to Acknowledge. 
+
+required to build the `jsbundle` and then compiles `Clojure` into `JavaScript`, watches for changes on `cljs` files, and hot-reloads code in the app.
+
+wait till you see the following message :
+
+```
+[:mobile] Build completed. (1801 files, 52 compiled, 0 warnings, 9.52s)
+```
+
+Once the clojure terminal is running you need to run the appropriate command next for your platform in a separate terminal :
+
+`make run-android` or `make run-ios`
+
+These commands will build the app, 
+
+start a metro bundler and deploy the app on your simulator OR connected device (android only).
+
+For building and deploying to connected iPhones use `make run-ios-device` instead of `make run-ios`
```

Comment:
Why are you splitting sentences like this?
```suggestion
These commands will build the app, start a metro bundler and deploy the app on your simulator OR connected device (android only). For building and deploying to connected iPhones use `make run-ios-device` instead of `make run-ios`
```

Reply:
My bad.
Updated it now.